### PR TITLE
[google_maps_flutter] Added LatLngBounds constructor from list of LatLng

### PR DESCRIPTION
**Pull request by tolotrasamuel**
I added LatLngBounds.fromLatLngList() constructor from list of LatLng.

Usage: 

```
List<LatLng> list = List<LatLng>();
list.add(LatLng(-20.10, 57.56);
list.add(LatLng(-20.12, 57.59);
list.add(LatLng(-20.15, 57.52);
LatLngBounds latlngbounds = LatLngBounds.fromLatLngList(list);
mapController
          .animateCamera(CameraUpdate.newLatLngBounds(LatLngBounds.fromLatLngList(list), 50.0));
   
```

### CODE REVIEW

**Comment by tolotrasamuel** [comment]
I signed it!

**Comment by RafaO** [comment]
I did it in a slightly different way here: https://github.com/flutter/plugins/pull/981 It was a while ago though. :D

File: packages/google_maps_flutter/lib/src/location.dart [hunk 1]
```diff
@@ -106,6 +106,23 @@ class LatLngBounds {
...
 
+  static LatLngBounds fromLatLngList(List<LatLng> list) {
+    assert(list.isNotEmpty);
+    double x0, x1, y0, y1;
+    for (LatLng latLng in list) {
+      if (x0 == null) {
+        x0 = x1 = latLng.latitude;
+        y0 = y1 = latLng.longitude;
+      } else {
+        if (latLng.latitude > x1) x1 = latLng.latitude;
```
Comment (domesticmouse) [inline packages/google_maps_flutter/lib/src/location.dart]: I think you forgot to handle the case of a latlng list that goes across the anti-meridian.

See Wikipedia's [180th_meridian](https://en.wikipedia.org/wiki/180th_meridian) for more info.

**Comment by cyanglaz** [comment]
@tolotrasamuel I assume it is a helper method for certain functionalities? I wonder if we should leave this to app developers (people who use the plugin) to implement on their own based on their needs.

Is there a Github issue you can link us to about this? It might help us to see how many people want this feature inside the plugin.

**Comment by stuartmorgan** [comment]
Thanks for the contribution! However, I'm closing this as a duplicate of https://github.com/flutter/plugins/pull/981 as we don't need two PRs for the same functionality.

**Pull request closed without merging**